test/functional/tools/version_command_plain.xml

```diff
@@ -1,6 +1,6 @@
 <tool id="version_command_plain" name="version_command_plain" version="1.0.0">
     <version_command><![CDATA[
-echo "4.0.0"
+VERSION="4.0.0" echo "$VERSION"
```

Comment:
This syntax only works if the command is not a shell internal but a separate process (e.g. `VERSION="4.0.0" bash -c 'echo "$VERSION"'`)

```suggestion
VERSION="4.0.0"; echo "$VERSION"
```